db/seeds/seedDatabase.ts

```
@@ -178,32 +178,21 @@ type MessageDetail = {
 const fixturesPath = path.join(dirname(fileURLToPath(import.meta.url)), "fixtures");
 const fixtureData = fs.readdirSync(fixturesPath).reduce<Fixtures>((acc, file) => {
   const content = JSON.parse(fs.readFileSync(path.join(fixturesPath, file), "utf8")) as Fixtures;
-  const [fixtureId, conversations] = Object.entries(content)[0]!;
-  return {
-    ...acc,
-    [fixtureId]: {
-      ...(acc[fixtureId] ?? {}),
-      ...conversations,
-    },
-  };
+  return { ...acc, ...content };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)